<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgTaskActivity</class>
 <widget class="QDialog" name="CDlgTaskActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activity</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTitle"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Date Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbSolar">
       <property name="text">
        <string>Solar</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbLunar">
       <property name="text">
        <string>Lunar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Start Time：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtStart">
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="tmStart">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>End Time：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtEnd">
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="tmEnd">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Repeat</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbOnce">
        <property name="text">
         <string>Once</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbEveryDay">
        <property name="text">
         <string>Every day</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbWeekly">
        <property name="text">
         <string>Weekly</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rbMonthly">
        <property name="text">
         <string>Monthly</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbEveryYear">
        <property name="text">
         <string>Every year</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rbCustom">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Prompt:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPromptAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPromptRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QListView" name="lstPromt"/>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Account:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAccount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Place:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePlace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Content:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QTextEdit" name="teContent"/>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel(&amp;C)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leTitle</tabstop>
  <tabstop>rbSolar</tabstop>
  <tabstop>rbLunar</tabstop>
  <tabstop>dtStart</tabstop>
  <tabstop>tmStart</tabstop>
  <tabstop>dtEnd</tabstop>
  <tabstop>tmEnd</tabstop>
  <tabstop>rbOnce</tabstop>
  <tabstop>rbEveryDay</tabstop>
  <tabstop>rbWeekly</tabstop>
  <tabstop>rbMonthly</tabstop>
  <tabstop>rbEveryYear</tabstop>
  <tabstop>rbCustom</tabstop>
  <tabstop>pbPromptAdd</tabstop>
  <tabstop>pbPromptRemove</tabstop>
  <tabstop>lstPromt</tabstop>
  <tabstop>leAccount</tabstop>
  <tabstop>lePlace</tabstop>
  <tabstop>teContent</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
